include/RAJA/util/View.hpp

```diff
@@ -296,6 +295,89 @@ RAJA_INLINE AtomicViewWrapper<ViewType, AtomicPolicy> make_atomic_view(
   return RAJA::AtomicViewWrapper<ViewType, AtomicPolicy>(view);
 }
 
+struct layout_left {
+};
+struct layout_right {
+};
+
+template <typename LAYOUT>
+struct Reshape;
+
+namespace detail
+{
+template <typename T>
+constexpr auto get_last_index(T last)
+{
+  return last;
+}
+
+template <typename T, typename... Args>
+constexpr auto get_last_index(T, Args... args)
```

Comment:
I think this should be like this so it can't be called with 1 argument.
```suggestion
template <typename T0, typename T1, typename... Args>
constexpr auto get_last_index(T0, T1, Args... args)
```

Reply:
I didn't fix the function call inside though